libsolidity/CompilerStack.cpp

```diff
@@ -179,6 +192,24 @@ string const& CompilerStack::getMetadata(string const& _contractName, Documentat
 	return *(*doc);
 }
 
+std::string const CompilerStack::getFunctionHashes(std::string const& _contractName)
```

Comment:
Agreed. It was the easiest place to put it in the beginning for testing but this will go somewhere else.